todocore/src/test/java/com/knighttodo/todocore/integration/DayResourceIntegrationTest.java

```diff
@@ -220,4 +223,15 @@ public void deleteDay_shouldDeleteDayWithAllDayTodos_whenIdIsCorrect() throws Ex
         assertThat(dayRepository.findByIdAlive(day.getId())).isEmpty();
         assertThat(dayTodoRepository.findAllAlive().isEmpty());
     }
+
+    @Test
+    @Transactional
+    public void findDayByDate_shouldReturnExistingDay_whenDateIsCorrect() throws Exception {
```

Comment:
what about negative scenarios? If it's null or empty?

Reply:
I added two tests for null and empty.